ambry-api/src/main/java/com.github.ambry/router/OperationTrackerScope.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.router;
+
+/**
+ * The metric scope applied to operation tracker. For now, the scope defines at which granularity the latency distribution
+ * should be tracked by operation tracker (i.e. ColoWide means latencies of all requests in local datacenter would be kept
+ * in a single Histogram)
+ */
+public enum OperationTrackerScope {
+  ColoWide, PartitionLevel
```

Comment:
How do you feel about changing the name of these values to `Datacenter` and `Partition`. I feel that the level and wide in the names is not needed and that datacenter better matches the terminology in the clustermap.

Reply:
Excellent point. Will take the suggestion. (Thus, we can avoid the term `Colo` which may confuse some people outside LinkedIn.)